recipe/meta.yaml

```diff
@@ -161,11 +159,12 @@ outputs:
         # optional dependencies for CUDA 11.2+
         - nccl ~=2.16                           # [build_platform != target_platform]
         - cutensor ~=2.0                        # [build_platform != target_platform]
+        - cusparselt ~=0.6.0
```

Comment:
Support for cuSPARSELT 0.6 was added in PR: https://github.com/cupy/cupy/pull/8074